cross/openzwave/digests

```diff
@@ -0,0 +1,3 @@
+openzwave-r1009.tar.gz SHA1 6acc7bd5878b0e68df9bfa139c004c035b2ccf73
```

Comment:
Fwiw, if you go with downloading the snapshot.tar.gz file, you should use the digests again ofc, it's just the svn checkout method which doesn't work with a digests file.